mappings/net/minecraft/util/random/RandomGenerator.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/util/random/RandomGenerator
+	COMMENT An interface describing random behavior in Minecraft.
+	COMMENT
+	COMMENT <p> This interface should be used instead of {@link java.util.Random} for random number generation.
+	METHOD m_afzjxjzq nextInt (I)I
+		ARG 1 bound
+	METHOD m_bggirwfo createPositionalRandomFactory ()Lnet/minecraft/unmapped/C_coqzjjrt;
+	METHOD m_culopwlx skip (I)V
+		ARG 1 count
+	METHOD m_djbqzhqs nextFloat ()F
+	METHOD m_ezprnxnd nextInt ()I
+	METHOD m_fakwfybt nextDouble ()D
+	METHOD m_ifdmmgky nextLong ()J
+	METHOD m_imwpekuz createUnsafe ()Lnet/minecraft/unmapped/C_rlomrsco;
+	METHOD m_iwfdczgh setSeed (J)V
+		ARG 1 seed
+	METHOD m_lzlwtgyn nextGaussian ()D
+	METHOD m_mkdxraps nextBoolean ()Z
+	METHOD m_ngqtmirh createLegacy (J)Lnet/minecraft/unmapped/C_rlomrsco;
```

Comment:
Is this [`createLegacy`] made to be consistent with other mappings elsewhere that are named the same? It doesn't seem clear enough to me as to what exactly it describes - I presume it is creating a legacy version of a random generator, but you might want to double-check and make sure it can't be confused between some different meanings:

- Creates an instance of RandomGenerator, with legacy options
- Creates an instance of a separate class, e.g. LegacyRandomGenerator (though I imagine that would be `createLegacyRandomGenerator`?)
- Creates a legacy/report of its current state
- Creates a 'legacy' entry in something that keeps track of the history of random generators

You can mostly rule these out logically, but I'm thinking that it has the possibility of being mistaken or confused, e.g. with beginners or just if someone's tired.  Do you think this is a valid concern, or does this meet other standards consistently enough for it not to be a worry?

Reply:
It's the second bullet. No need for the full name however.